<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnaxelCreatorWidgetUI</class>
 <widget class="QWidget" name="SnaxelCreatorWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Create front</string>
   </property>
  </widget>
  <widget class="QSlider" name="goalNVSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="goalNVSpinBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Goal N*V: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
